src/fstrm_capture.c

```diff
@@ -163,6 +163,12 @@ static argv_t g_args[] = {
 		"<PORT>",
 		"TCP socket port to read from" },
 
+	{ 'b',	"buffer_size",
+		ARGV_INT,
+		&g_program_args.buffer_size,
+		"<SIZE>",
+		"Frame streams read buffer size, in bytes" },
```

Comment:
Should be "Frame Streams" rather than "Frame streams"

Reply:
Fixed in 9e3827aaeb1c79270797be4a67510a5910f35f73